### PR TITLE
update done UC

### PR DESCRIPTION
**Pull request by absurd-turtle**
## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

File: doc/usecases/done/login.md [hunk 1]
```diff
@@ -27,3 +27,9 @@
 - **Non-functional Constraints:**
   - The login process must be secure to protect the user's personal information.
   - The app must clearly communicate to the user whether the login was successful or not.
+
+## Links
+
+[Authentication Setup](../../research/nextcloud_integration.md)
+[Authentication Decision](../../decisions/auth.md)
+
```
Comment (markus2330) [inline doc/usecases/done/login.md:35]: Formatting is broken. And please also write in Changelog that UC are done (maybe to the release when they were actually done).

**Review by markus2330 — APPROVED** [review]
Great job but please fix CI and add to changelog.

**Comment by absurd-turtle** [comment]
There is an md book error about the links.
What should we do about it?

**Comment by markus2330** [comment]
I would prefer if we allow to link all files in / (instead of /doc), so that we can link to code etc.

As quick-fix we can simply use `code` instead of a link.

**Comment by markus2330** [comment]
Ok, I see it is unfortunately currently impossible to have links outside of the `doc` folder for us (mdbook only allows one folder as root, doesn't allow excludes, and cannot link outside of the root folder). So let us go with normal web links, I hopefully fixed it.

**Pull request merged**